code/modules/admin/holder2.dm

```diff
@@ -232,7 +232,7 @@ GLOBAL_PROTECT(href_token)
 		return VALID_2FA_CONNECTION
 
 	if (!SSdbcore.Connect())
-		if (verify_backup_data(client))
+		if (verify_backup_data(client) || client.ckey in GLOB.protected_admins)
```

Comment:
![image](https://github.com/tgstation/tgstation/assets/34697715/9964c42d-6605-4fd8-a5db-a5eae64d6468)


```suggestion
		if (verify_backup_data(client) || (client.ckey in GLOB.protected_admins))
```

thank you dreamchecker very cool